packages/esm-active-visits-app/src/config-schema.ts

```diff
@@ -18,31 +19,64 @@ export interface IdentifiersDefinition {
   identifierName: string;
 }
 
+export interface AttributeDefinition {
+  display: string;
+  header: {
+    key: string;
+    default: string;
+  };
+}
+
 export const configSchema = {
   activeVisits: {
     identifiers: {
       _type: Type.Array,
       _description: 'Customizable list of identifiers to display on active visits table',
       _elements: {
+        _type: Type.Object,
```

Comment:
No, this is not what `Type.Object` is for. `Type.Object` is for unstructured objects. It works like `Type.Array`, but for objects. It should be avoided when possible, in favor of `Type.Array`.

I almost wonder if we should deprecate it, since one can always define an array config that achieves the same thing:

```
_type: Type.Array,
_elements: {
  key: { _type: Type.String },
  value: { _type: Type.String }
}
```